### PR TITLE
US 162

### PR DESCRIPTION
**Pull request by JeffFallis**
## Objective
Adding Home page recreation in React

## Detailed Description
Added home page. 

## Screenshot (if applicable)
image

## How to Test (if applicable)
Build containers and go to
localhost:8001

## Requirements
https://tree.taiga.io/project/cbrobsto-iron-db/us/162?milestone=254780
https://tree.taiga.io/project/cbrobsto-iron-db/task/191
https://tree.taiga.io/project/cbrobsto-iron-db/task/244

### CODE REVIEW

File: API/public/javascripts/profile.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ function validatePassword() {
         }
 
         if (!hasLowerCase) {
-          $('.passwordError').text('Password must contain at least 1 lower case character!');
+          $('.passwordError').text('Password must contain at least lower case character!');
```
Comment (mjidris) [inline API/public/javascripts/profile.js]: Should we keep the `1` in?

**Review by mjidris — APPROVED** [review]
💯

**Review by StellaJaneRasmussen — APPROVED** [review]
Looks great! Thanks for getting this set up.

File: API/public/javascripts/profile.js (hunk 1 above)
Comment (JeffFallis) [reply API/public/javascripts/profile.js]: You are right. I'm not sure how I did not see that.

Comment (JeffFallis) [reply API/public/javascripts/profile.js]: Added a commit to fix it

**Pull request merged**